editor/src/clj/editor/app_view.clj

```diff
@@ -294,6 +296,81 @@
                                 (get sub-selections-by-resource-node active-resource-node)))
   (output refresh-tab-panes g/Any :cached (g/fnk [^SplitPane editor-tabs-split open-views open-dirty-views]
                                             (let [tab-panes (.getItems editor-tabs-split)]
+
+                                              ;TODO: Extract this logic to another method.
+                                              (let [total-tabs (reduce + (map #(-> ^TabPane % (.getTabs) (.size)) tab-panes))
```

Comment:
@matgis @britzl 
I really want to extract this code to another function.

I have try create a function (in this file) like this:

```
(defn- update-quick-help-pane [arg]
; implement here
)
```

But I don't know how to call this method from here.
Could you please suggest?

Thx!